### DISCUSSION TITLE
reaction vs direct use of the ratom

### DISCUSSION THREAD

**Discussion by ShayMatasaro** [day8/re-frame#703] [Q&A]
Hi,

Thanks for the great article.

What is the advantage of using a reaction over a direct access to the ratom, for example why is : 
[:div  (reaction (:a @app-db))]

better then : 
[:div (:a @app-db)]

Either way when app-db changes it will trigger an evaluation of of the div .

Is there a better place to post questions ? are you on IRC/chat?

Cheers,
Shay

**Comment by mike-thompson-day8** [comment]
`reaction` should not be used in views (as you have done above).  Should only be done in subscriptions.  Best to have a look at the examples.

**Comment by mike-thompson-day8** [comment]
The two best place to ask questions are:
1.  http://clojurians.net/  re-frame channel
2.  groups  https://groups.google.com/forum/#!forum/clojurescript
